ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/router-bgp.j2

```diff
@@ -34,7 +34,12 @@
 | Settings | Value |
 | -------- | ----- |
 {%             if router_bgp.peer_groups[peer_group].type is arista.avd.defined %}
-| Address Family | {{ router_bgp.peer_groups[peer_group].type }} |
+{%                 if router_bgp.peer_groups[peer_group].type is not string  %}
```

Comment:
This `type` variable is only used in documentation and in some check in the validation role. Please don't introduce all this logic around it. Just use it as a string in your molecule. Ex. `type: "mpls"` or `type: "vpnv4, vpnv6"`

Reply:
Removed it as per suggestion.